### ISSUE TITLE
Font size 

### ISSUE THREAD

**Issue by M0lyk** [SecUSo/privacy-friendly-2048#12]
Hi, thank you so much for this game.
I'm having an issue with font size with the tile 16384 : 
![Screenshot_20200305-083109](https://user-images.githubusercontent.com/1733940/75958304-35efec00-5ebc-11ea-82e5-960dc78dea1f.png)

**Comment by alexandrestein** [comment]
Even on regular grid it happened to me.
It started small and decreased over time to become invisible.

At the end of the game it reappeared like on the screenshot above.
![Screenshot_20211002-105514](https://user-images.githubusercontent.com/5431025/135710104-2dd14dc3-f8cc-40fe-84c7-9cacae2af76d.png)

Everything else is great.
Thanks 😊

**Comment by killerbees19** [comment]
16384 font gets smaller with each movement 😅

But the font size resets when rotating the screen or even taking a screenshot.

**Comment by inson1** [comment]
Yes, it get smaller and after screenshot this happnens ![Screenshot_2024-02-22-08-53-36-773_org.secuso.privacyfriendly2048.jpg](https://github.com/SecUSo/privacy-friendly-2048/assets/75314629/c4599b36-c621-49d5-a71d-635356b24116)

text is too long and cant fit, soI cant see last 8

**Comment by inson1** [comment]
32768 has is it too

**Comment by mhastu** [comment]
For me it already happens with 1024, supposedly because of "Large" font in Android settings.
![Screenshot_20240518-114129_2048](https://github.com/SecUSo/privacy-friendly-2048/assets/109336419/439414da-5a20-4860-a5fa-c41d8c31a422)
